From a4cecfcf8008704c9ff8f447adf1158b426b149d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jaime=20Marqui=CC=81nez=20Ferra=CC=81ndiz?=
 <jaime.marquinez.ferrandiz@fastmail.net>
Date: Thu, 29 May 2025 23:48:18 +0200
Subject: [PATCH] Fix build error with the -Werror=return-type flag

This flag is automatically enabled in openSUSE Tumbleweed.
---
 src/librssguard-standard/src/standardfeed.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/librssguard-standard/src/standardfeed.cpp b/src/librssguard-standard/src/standardfeed.cpp
index fb0da2b8c..7a52ba1a3 100644
--- a/src/librssguard-standard/src/standardfeed.cpp
+++ b/src/librssguard-standard/src/standardfeed.cpp
@@ -480,6 +480,9 @@ QString StandardFeed::getHttpDescription() const {
 
     case NetworkFactory::Http2Status::Disabled:
       return tr("disabled");
+
+    default:
+      return tr("unknown state");
   }
 }
 
